pkg/cnab/provider/credentials_test.go

```diff
@@ -145,3 +146,42 @@ func TestRuntime_loadCredentials_WithApplyTo(t *testing.T) {
 	})
 
 }
+
+func TestRuntime_nonSecretValue_loadCredentials(t *testing.T) {
+	t.Parallel()
+
+	r := NewTestRuntime(t)
+	defer r.Close()
+
+	b := cnab.NewBundle(bundle.Bundle{
+		Credentials: map[string]bundle.Credential{
+			"password": {
+				Location: bundle.Location{
+					EnvironmentVariable: "PASSWORD",
+				},
+			},
+		},
+	})
+
+	run := storage.Run{
+		Action: cnab.ActionInstall,
+		Credentials: storage.NewInternalCredentialSet(secrets.SourceMap{
+			Name: "password",
+			Source: secrets.Source{
+				Strategy: host.SourceValue,
+				Hint:     "mypassword",
+			},
+		}),
+	}
+	err := r.loadCredentials(context.Background(), b, &run)
```

Comment:
Think we could make a testcontext and pass that in here, but we don't have to (the func is like `NewPorterTestContext` or `NewTestContext` ?)